SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
```

Comment:
I believe all Solidity versions are supposed to get security updates.

Reply:
I don't think so. Only the latest active tree gets updates. In exceptional circumstances, when a breaking release is very young (less then 3-4 releases in), we did issue security updates to the older tree.

Reply:
Yeah, it would be too large of a maintenance burden to support _all_ versions. At most we could have some LTS versions, but that probably only starts making sense once we reached v1.0

Reply:
Ah I see, I remember some security fixes getting back ported as far back as `0.4.x` but perhaps that was one off?

> Only the latest active tree gets updates

Does this mean the version in which the bug was found and backwards until the last breaking release. So for example, if bug was found in `0.6.2`, then `0.6.0` up to `0.6.2` get patched?

Reply:
No, just a new release is made in the tree. E.g. since we are early in 0.6.x, it may make sense to make a 0.5.x bugfix release. I think however these cases are very rare if keep our release schedule, because people will transition to new breaking releases more quickly (that's the hope).

The 0.4.x vs 0.5.x was a special case, because 0.4.x was so widely used and it took a really long time to release 0.5.x